test/pathToParams.test.js

```diff
@@ -0,0 +1,81 @@
+const pathToParams = require("../util/pathToParams.js");
+
+describe("pathToParams", () => {
+  const ALLOWED_EXTENSIONS = ["jpeg", "png", "webp", "gif", "svg", "jfif"];
+  describe("Given a URL with both dimensions and no output-format", () => {
+    it("Will extract dimensions and use the original extension as outputFormat", () => {
+      const input = "500x320/foo.jpg";
+      const [size, path, outputFormat, originalExtension] = pathToParams(
+        input,
+        ALLOWED_EXTENSIONS
+      );
+      expect(size).toBe("500x320");
+      expect(path).toBe("foo.jpg");
+      expect(outputFormat).toBe("jpeg");
+      expect(originalExtension).toBe("jpg");
+    });
+  });
+  describe("Given a URL with just a width", () => {
+    it("Will still extract the dimensions", () => {
+      const input = "/500x/foo.jpg";
+      const [size, path, outputFormat, originalExtension] = pathToParams(
+        input,
+        ALLOWED_EXTENSIONS
```

Comment:
I would replace it with a simple array to prevent confusion. To me it looks like it has something to do with the tests, but it doesn't. And it decouples the test, although that's a very small issue.

```suggestion
        ['jpg']
```

Reply:
Fair enough! Will do.